lib/big_query/base/query.ex

```diff
@@ -11,6 +11,16 @@ defmodule BigQuery.Base.Query do
     run_query(queryParams, sql, pageLimit)
   end
 
+  def query_each(sql, func), do: query_each(%{}, sql, func)
+  def query_each(params, sql, func), do: query_each(params, sql, nil, func)
+  def query_each(params, sql, limit, func) do
```

Comment:
So rather than running the query and returning `{results, meta}` ... this just returns `meta`, and calls the function for every page of result rows.  You can explicitly add a per-page `limit`, but in all the rollups they just leave it nil, and BQ gets to decide the max payload size.